### ISSUE TITLE
colon_assignment_spacing seems buggy

### ISSUE THREAD

**Issue by evanworley** [clutchski/coffeelint#208]
When I configure colon_assignment_spacing with left=0, it doesn't seem to work correctly.

``` json
  "colon_assignment_spacing": {
    "level": "error",
    "spacing": {
      "left": 0,
      "right": 1
    }
  }
```

For the line

``` coffeescript
      timestamp: notification.data.transfer.created
```

Colon assignment without proper spacing. `Expect colon spacing left: 0, right: 1. Got left: 1, right: 2..` is reported. It doesn't look like i have one space to the left of the colon, nor two spaces to the right. Off by one error?
